cmd/knit/subcommands/run/find/command_test.go

```diff
@@ -173,26 +184,47 @@ func TestFindCommand(t *testing.T) {
 		},
 	))
 
-	t.Run("when values for each flag are passed, it should call task with these values", theory(
+	{
+		timestamp := time.Date(
+			2024, 4, 22, 0, 00, 00, 000000000,
+			time.FixedZone("+09:00", int((9*time.Hour).Seconds())),
+		)
```

Comment:
I suppose that these 2 or more zeroes are expressing "this is the minute", "this is the second" or "this is the subsecond". But, it is not encouraged because

- [Numbers following 0 is octal](https://go.dev/ref/spec#Integer_literals), so `09` can not be written and does not cover all time expression.
- If it is difficult to read a row of arguments, you should use more readable expression, RFC3339, like below

```suggestion
		timestamp := try.To(rfctime.ParseRFC3339DateTime("2024-04-22T00:00:00.000+09:00")).OrFatal(t).Time()
```

By using RFC3339, we do not need to count arguments in `time.Date`.